types/newman/index.d.ts

```diff
@@ -173,10 +173,14 @@ export interface NewmanRunExecutionAssertionError {
     test: string;
     message: string;
     stack: string;
-}export interface NewmanRunFailure {
+}
+export interface NewmanRunFailure {
     error: NewmanRunExecutionAssertionError;
     /** The event where the failure occurred */
     at: string;
+    source: any;
```

Comment:
My best estimate for this
```suggestion
    source: NewmanRunExecutionItem | undefined;
```

Reply:
@Graham42 Should it be `source?: NewmanRunExecutionItem`

Reply:
Not arguing btw, just want to know what the best practice/style is with typescript.